bears/haskell/HaskellLintBear.py

```diff
@@ -3,8 +3,8 @@
 from coalib.bearlib.abstractions.Linter import linter
```

Comment:
also body shouldnt be in first person, try something like: PyImportSortBear was used to sort the imports correctly, could you :D ?